### DISCUSSION TITLE
Newer uncensored models

### DISCUSSION THREAD

**Discussion by sussyboiiii** [ggml-org/llama.cpp#4443] [Q&A] [answered]
I have been searching for a "truly" uncensored model like the first llama models or alpaca models if im not mistaken have been. If you ask them some weird stuff they wont say its unethical they'll just reply as normally they would.

**Comment by supportend** [comment]
TheBloke has many models.

https://huggingface.co/TheBloke

mistralai_mixtral-8x7b-instruct-v0.1 never refused answers for me, but sometimes it means, a answer is not possible, like the last 10 digits from pi. This is a feature, i like. I asked, "What is llama.cpp?". Other models wrote "A ChatGPT client..." and other fantasy stuff.

**Reply by sussyboiiii** [reply]
Ok but if I asked it on how to make meth at home would it give me a reply or just say no it's bad don't?

**Reply by supportend** [reply]
I never ask such questions. On HF are many Models with "uncensored" in the model names, download them, try them? For example, when you expand all models from TheBloke a search in the site "uncensored" gives many results. Read the comments to decide, a download makes sense for you.

**Reply by sussyboiiii** [reply]
Yes ill take a look. Mixtral doesnt run on my system anyways. Thank you!

**Reply by supportend** [reply]
Someone works on 1-Bit Q, without affecting the quality much:
https://github.com/ggerganov/llama.cpp/issues/4445

Mixtral Instruct is the best model, i tested. But i test it in German mosty. It's much faster than 70B models on my system.

**Reply by sussyboiiii** [reply]
If this will be implemented itll be really interesting. Is it good at understaning german and also context?

**Reply by supportend** [reply]
Yes, i run it with ctx 32768 and use the server - web gui. I don't have a good GPU, it runs fast enough on my CPU, but the Q6 gguf consums 39,9 GiB RAM at the moment.

**Reply by sussyboiiii** [reply]
Theres a web gui?

**Reply by supportend** [reply]
Yes, start server instead of main and look at the output.

Example:
`./server -t 8 -m ~/Downloads/models/mixtral-8x7b-instruct-v0.1.Q6_K.gguf -c 32768 --log-disable`

On my system i open http://127.0.0.1:8080/ in my web browser. In the settings of the gui are many options.

**Reply by sussyboiiii** [reply]
ive seen the ui sort of cool, im still waiting for someone to make a gui to run as a normal application in which i can use any model in gguf format and also have things like code highliting and stuff

**Reply by supportend** [reply]
GPT4All can handle guuf too, but i'm happy with llama.cpp - server and main. I saw other GUIs in the model cards from TheBloke, i don't know, what they can and what not. In the server gui from llama.cpp i see light text with dark background, when it adds code, this is enough for me, because i paste it in cpp, py files directly.

**Reply by sussyboiiii** [reply]
thanks for the gpt4all recommendation, ive tested mistral instruct 7B vs wirzardlm 13B and i mean its not really fair but im impressed that wizard managed to solve my problem in coding first try.

`﻿﻿﻿Write python code which opens a winow with a button in it which when pressed changes the background color to a random color`

**Reply by supportend** [reply]
I used a model from phind before, it was able to generate a c++ program with up to 4 parameters very well. mixtral-8x7b-instruct-v0.1 wrote me a program, to display cpu and gpu temperature or one of them (-cpu, -gpu, -beide). But i recommend phind.com for programming, because it is able to find and combine information in the recent web. (I don't use ChatGPT, because i don't have a smartphone, this is required, to register on OpenAI)

**Reply by sussyboiiii** [reply]
That's pretty cool but sadly my 16GB M1 Pro is struggling running these models, thanks for the phint recommendation though!

**Reply by londex033** [reply]
Quelle est la capitale de la France ?

**Reply by sussyboiiii** [reply]
At least write you question in English

**Reply by Alumniminium** [reply, marked as answer]
Spicyboros, highly recommended, it has function calling and context based answering, thanks to airoborus dataset training. very useful for multiple use cases. will even cosplay hitler if you ask it to.

**Reply by sussyboiiii** [reply]
Thank you man, this model really doesn't care. Exactly what I've been searching for!

Sometimes it gets side tracked though do you mind sharing your prompt?

**Reply by Alumniminium** [reply]
https://huggingface.co/TheBloke/Spicyboros-c34b-2.2-GGUF?not-for-all-audiences=true#prompt-format
please check out all the supported prompt templates, depending on what you want to achieve, you have to use different prompts formats.

**Reply by sussyboiiii** [reply]
Thanks!
